rdtools/normalization.py

```diff
@@ -76,17 +77,17 @@ def pvwatts_dc_power(poa_global, P_ref, T_cell=None, G_ref=1000, T_ref=25,
     ----------
     poa_global : pd.Series
         Total effective plane of array irradiance.
-    P_ref : float
+    pmp_ref : float
         Rated DC power of array in watts
-    T_cell : pd.Series, optional
+    temperature_cell : pd.Series, optional
         Measured or derived cell temperature [degrees celsius].
         Time series assumed to be same frequency as `poa_global`.
         If omitted, the temperature term will be ignored.
-    G_ref : float, default 1000
+    irradiance_ref : float, default 1000
         Reference irradiance at standard test condition [W/m**2].
-    T_ref : float, default 25
+    temperature_ref : float, default 25
         Reference temperature at standard test condition [degrees celsius].
-    gamma_pdc : float, default None
+    gamma_pmp : float, default None
```

Comment:
`gamma_pmp_ref`?

Reply:
@cwhanse @wholmgren I see that gamma_pdc is used in pvlib, but is listed as deprecated in pv-terms. Is there a plan to change to gamma_pmp_normalized in pvlib?

Reply:
I don't have any plans to change it.

Reply:
Agree with @wholmgrem here

Reply:
I think it makes sense to stay aligned with pvlib